### PR TITLE
DOC Staticpublishqueue changelog entry

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/100

Changelog entry for https://github.com/silverstripe/silverstripe-staticpublishqueue/pull/168

### CODE REVIEW

**Comment by emteknetnz** [comment]
@chrispenny could I get you to check you're happy with this

File: en/04_Changelogs/5.1.0.md [hunk 1]
```diff
@@ -119,6 +120,18 @@ SilverStripe\SessionManager\Models\LoginSession:
...
+### Static publish queue related page regeneration {#staticpublishqueue-regeneration}
+
+New configuration options have been added to [Static Publish Queue](https://github.com/silverstripe/silverstripe-staticpublishqueue) to force regeneration of related pages after publishing or unpublishing a related page. You can enable this with the following configuration:
+
+```yml
+SilverStripe\CMS\Model\SiteTree:
+  regenerate_children: recursive
+  regenerate_parents: recursive
+```
+
```
Comment (chrispenny) [inline en/04_Changelogs/5.1.0.md:132]:
```suggestion

Available options are:

- `none`: Do not regenerate any parent or child hierarchy
- `direct`: Regenerate only one level above or below (direct parent or children, but not grandparent or grandchildren)
- `recursive`: Regenerate the entire parent or child hierarchy

```

**Review by chrispenny — COMMENTED** [review]
Looks good to me. If you'd like to include the possible options in this changelog, then I've added a suggestion. I think you've linked to the full docs though, which is also fine.

File: en/04_Changelogs/5.1.0.md (hunk 1 above)
Comment (emteknetnz) [reply en/04_Changelogs/5.1.0.md:132]: Updated

**Review by GuySartorelli — APPROVED** [review]
Looks okay to me.

**Pull request merged**